modules/ROOT/pages/policies-policy-overview.adoc

```diff
@@ -0,0 +1,148 @@
+= Policies Overview
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+//where are we covering what a policy? What the policy does
+
+Policies differ based on several different factors, such as level of granularity,types of policies, and policy categories. You must carefully review these factors, before you decide to implementation them in your environment. In this section, each of these factors are explained in detail. 
```

Comment:
I think the factors should be: category, purpose, version and configuration options.

Reply:
Done! Thanks!